app/models/rbac.rb

```diff
@@ -222,7 +222,7 @@ def self.find_targets_filtered_by_parent_ids(parent_class, scope, find_options,
   end
 
   def self.find_targets_filtered_by_ids(scope, find_options, u_filtered_ids, filtered_ids)
-    total_count  = total_scope(scope, u_filtered_ids, find_options[:conditions], find_options[:include]).count
+    total_count  = total_scope(scope, u_filtered_ids, find_options[:conditions], find_options[:include]).count(:all)
```

Comment:
This is the change to Rbac that doesn't have a PR to reference.  You can try and find one but I gave up after a while.  This is the line I stole from master where the `.count(:all)` comes from:

https://github.com/ManageIQ/manageiq/blob/e0aa649/lib/rbac/filterer.rb#L222